app/enterprise/0.35-x/deployment/installation/centos.md

```diff
@@ -1,216 +1,238 @@
+
+1. Make a copy of the default configuration file
 
-10. Make a copy of the default configuration file    
     ```
     $ cp /etc/kong/kong.conf.default /etc/kong/kong.conf
     ```
 
-11. Uncomment and add 'kong' to pg_password line
+2. Uncomment and update the PostgresQL database properties inside the Kong conf:
+
     ```
     $ sudo vi [/path/to/kong.conf]
     ```
+    ```
+    pg_user = kong
+    pg_password = kong
+    pg_database = kong
+    ```
 
-## Step 6 
 
-Seed the **Super Admin** for initial startup. This step is not mandatory, 
-but if you want to add security and Role-Based Access Control (RBAC) to your 
-environment, it is best to seed the **Super Admin** at this step and make note 
-of the password for when you need it later. 
+## Step 6. Seed the **Super Admin** *(optional)*
 
-Create an environment variable with the _Super Admin_ password (that will be 
-used during migrations to seed the initial _Super Admin_ password):
+For added security and Role-Based Access Control (RBAC), it is best to seed 
+the **super-admin** before initial start up.
 
-```
-$ export KONG_PASSWORD=<password-only-you-know>
-```
+Create an environment variable with the desired **super-admin** password:
 
-## Step 7
 
-Run migrations to prepare the Kong DB
+    $ export KONG_PASSWORD=<password-only-you-know>
 
-```
-$ kong migrations bootstrap -c /etc/kong/kong.conf
-```
 
-## Step 8
+This will be used during migrations to seed the initial **super-admin** 
+password within Kong.
 
-Start Kong.
 
-```
-$ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
-```
+## Step 7. Start Kong
 
-## Step 9
+1. Run migrations to prepare the Kong database
 
-Verify Kong installation is working.
+    ```
+    $ kong migrations bootstrap -c /etc/kong/kong.conf
+    ```
 
-```
-curl -i -X GET --url http://localhost:8001/services
-```
+2. Start Kong
 
-## Next Steps
+    ```
+    $ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
+    ```
+
+3. Verify Kong is working
+
+    ```
+    curl -i -X GET --url http://localhost:8001/
+    ```
+
+
+## Troubleshooting
 
-You should receive a HTTP/1.1 200 OK message. If you do, you are ready to move on. As a Next Step, consider the following tasks:
+If you did not receive an HTTP/1.1 200 OK message, or need assistance completing
+setup reach out to your **Support contact** or head over to the
+[Support Portal](https://support.konghq.com/support/s/).
+
+
+## Next Steps
 
-1. Turn on RBAC and understand the RBAC model within Kong. 
-2. Turn on the Admin GUI Portal to allow you to administer Kong through the UI. 
-3. Create a sample Workspace to understand how Kong can be segregated for use between teams. 
-4. Create Services, Routes, and Consumers for a given workspace. 
-5. Add Plugins to inject functionality to the API calls that are being proxied through Kong. For example, Add Rate-Limiting. 
+Work through Kong's series of 
+[Getting Started](/enterprise/latest/getting-started) guides to get the most
```

Comment:
Why use `latest` instead of `{{kong.page_version}}`?

Reply:
Just a future-proofing decision incase it doesn't get updated each release